dev/ts/component/DropFileUserInterface.ts

```diff
@@ -153,7 +153,11 @@ export class DropFileUserInterface {
         const files = dt.files;
         dropCallback(files[0]);
       } catch (e) {
-        logErrorAndOpenAlert(e);
+        if (e instanceof Error) {
+          logErrorAndOpenAlert(e.message);
+        } else {
+          console.error('An error occurred:', e);
+        }
```

Comment:
**nitpick:** the error needs to be shown to the user, with the current proposal it would be hidden.

If `e` is not an Error, we could convert it as a string and pass it to the `logErrorAndOpenAlert` function.
Or even better, the processing could be done directly in the `logErrorAndOpenAlert` function.

Reply:
`e` is type of any. So we don't know how to convert it in `string` type :slightly\_frowning\_face:

Reply:
`String(e)` should to the trick, like done by `console.info(e)`